### PR TITLE
fix: adjust sound plugin

### PR DESCRIPTION
**Pull request by tsic404**
remove old v20 code, adjust sound devices widget style
run dbus call in another thread to prevent main thread get into wait.
make deviceList item undrageble, and sound setting clickable.

### CODE REVIEW

**Comment by tsic404** [comment]
Resolve: https://github.com/linuxdeepin/developer-center/issues/4247
Resolve: https://github.com/linuxdeepin/developer-center/issues/4218

**Review by Decodetalkers — DISMISSED** [review]

File: plugins/sound/sounddeviceswidget.cpp [hunk 1]
```diff
@@ -22,13 +23,16 @@
 #include <QProcess>
 #include <QDBusInterface>
 #include <QDBusConnection>
+#include <QtConcurrent/QtConcurrent>
```
Comment (BLumia) [inline plugins/sound/sounddeviceswidget.cpp:26]: CMakeLists 里漏找了 QtConcurrent 吧？这里 include 路径应该不对。

File: plugins/sound/sounddeviceswidget.cpp [hunk 2]
```diff
@@ -363,7 +357,7 @@ void SoundDevicesWidget::resizeHeight()
     }
     m_deviceList->setFixedHeight(deviceListHeight);
     QMargins m = layout()->contentsMargins();
-    int height = m.top() + m.bottom() + HEADERHEIGHT + m_sliderContainer->height() + ITEMSPACE
+    int height = m.top() + m.bottom() + HEADERHEIGHT + m_sliderContainer->height()  + ITEMSPACE
```
Comment (dengbo11) [inline plugins/sound/sounddeviceswidget.cpp:360]: 多了个空格,格式化下代码

File: plugins/sound/sounddeviceswidget.cpp [hunk 3]
```diff
@@ -431,10 +425,12 @@ void SoundDevicesWidget::onSelectIndexChanged(const QModelIndex &index)
         }
     } else {
         // 如果是点击声音设置，则打开控制中心的声音模块
-        DDBusSender().service("org.deepin.dde.ControlCenter1")
+         QFuture<void> f1 =QtConcurrent::run([=] {
```
Comment (Decodetalkers) [inline plugins/sound/sounddeviceswidget.cpp:428]: 这个f1好像没有地方使用诶，是我的错觉吗

Comment (tsic404) [reply plugins/sound/sounddeviceswidget.cpp:428]: 是没地方用到

**Review by Decodetalkers — APPROVED** [review]

**Pull request merged**